Add export tickets to fabric ticketing

ssalinas24 opened the pull request:
Jira: https://rackspace.atlassian.net/browse/PLATEXP-140

This PR adds Fabric Ticketing documentation for the  Export Tickets feature.

ssalinas24 commented:
@asmita32675 Please review this PR when you have a chance. Leave comments for any lines that I should fix or update. Thank You!

asmita32675 commented on content/how-to/rackspace-fabric/fabric-ticketing/index.md:
```diff
@@ -64,7 +64,23 @@ The user must be granted ticketing permissions for the ticket navigation items t
 
 4. Filter: This filter option can filter your tickets using options such as Device, Created By, Accounts, and Date Range 
-5. Refresh List: When you click on Refresh List option, the most recently created ticket appears on the top of the ticket list. 
+5. Refresh List: When you click on Refresh List option, the most recently created ticket appears on the top of the ticket list.
```
Make Refresh List as bold, since this is a button on the UI.

asmita32675 commented on content/how-to/rackspace-fabric/fabric-ticketing/index.md:
```diff
@@ -64,7 +64,23 @@ The user must be granted ticketing permissions for the ticket navigation items t
+5. Refresh List: When you click on Refresh List option, the most recently created ticket appears on the top of the ticket list.
+
+##### Export Tickets
+<img class="fig-img" src="/support/how-to/fabric-ticketing/export-tickets_01.png" alt="An expanded drop-down showing the export ticket and close ticket options">
```
Need to change "Close Tickets" to "Close Selected". As we click on Actions button, we could see in the drop down list Export Tickets and Close Selected. So our nomenclature in document should match with UI.

asmita32675 commented on content/how-to/rackspace-fabric/fabric-ticketing/index.md:
```diff
@@ -64,7 +64,23 @@ The user must be granted ticketing permissions for the ticket navigation items t
+
+2. Select to export either **Closed Tickets**, **Open Tickets**, or **Pending Tickets**.
+3. Select the **Export Tickets** button.
+4. A .csv file will download in the browser.
```
Need to delete period and csv should be CSV.

asmita32675 commented on content/how-to/rackspace-fabric/fabric-ticketing/index.md:
```diff
@@ -64,7 +64,23 @@ The user must be granted ticketing permissions for the ticket navigation items t
+
+**Warning**: Currently it is only possible to export a maximum of 5,000 tickets for each option.
 
 #### **Right Side Ticket Panel**:  
```
Need to delete colon from the heading. Also same change for Left Side Ticket Panel.

ssalinas24 replied:
@asmita32675  If you search the code base for `Warning`, you will notice that tons of files have the colon following it. However, it is sometimes within the bold like this `**Warning:**` and some times it is outside the bold like this `**Warning**:`. So I figured we needed it.

asmita32675 replied:
Agree.

asmita32675 commented on content/how-to/rackspace-fabric/fabric-ticketing/index.md:
```diff
@@ -64,7 +64,23 @@ The user must be granted ticketing permissions for the ticket navigation items t
+To **Export Tickets** do the following:
+1. Select **Actions** > **Export Tickets**.
+
+<img class="fig-img" src="/support/how-to/fabric-ticketing/export-tickets_02.png" alt="Export ticket popover showing radio button options to export Open Tickets, Closed Tickets, or Pending Customer tickets">
```
Change tickets to Tickets

ssalinas24 replied:
@asmita32675 `Open Tickets` and `Closed Tickets` are the actual titles of the options. The third option is titled `Pending Customer`, so I left the following `tickets` word lowercase. Do you still feel I need to capitalize `tickets`?

asmita32675 commented on content/how-to/rackspace-fabric/fabric-ticketing/index.md:
```diff
@@ -64,7 +64,23 @@ The user must be granted ticketing permissions for the ticket navigation items t
+
+<img class="fig-img" src="/support/how-to/fabric-ticketing/export-tickets_02.png" alt="Export ticket popover showing radio button options to export Open Tickets, Closed Tickets, or Pending Customer tickets">
+
+2. Select to export either **Closed Tickets**, **Open Tickets**, or **Pending Tickets**.
```
**Pending Customer** instead  **Pending Tickets** as per UI

ssalinas24 commented:
@asmita32675 I have addressed the feedback. Let me know if I need other changes. If everything looks good, then I just need your approval. You would need to click the green Review button on the top right of page, and select the approve radio option when submitting the review.
